functions.php

```diff
@@ -24,11 +24,11 @@
 	'/custom-comments.php',                 // Custom Comments file.
 	'/jetpack.php',                         // Load Jetpack compatibility file.
 	'/wp-custom-fields.php',                // Disable WP Core custom fields metaboxes.
-	'/advanced-custom-fields.php',          // Load integrated ACF plugin.
 	'/class-wp-social-media-walker.php',    // Load custom WordPress nav walker for footer Social Media menu wdiget.
 	'/wp-custom-menu.php',                  // Load custom menu builder functions.
 	'/editor.php',                          // Load Editor functions.
 	'/deprecated.php',                      // Load deprecated functions.
+	'/tgm-plugin-activation.php',            // Load TGM Plugin Activation script for dependent plugin recommendations.
```

Comment:
The order of these files has some significance. There's no hard rules necessarily, except that some later include files depend on functions being already loaded prior to them. And baseline setup of the theme occurs early in the order. I would suggest moving the tgm activation line up in the order, and since it is effectively replacing the old acf line, to move it into the same position. Unless there is a specific reason to load it late. :)

Reply:
Good call. That makes sense to me. :-)

Reply:
Although, if installing the Pro plugin does clear the notice, then maybe we should make the notice undismissable.

Reply:
Added code to move the declaration up to the previous spot where ACF was loaded.

Reply:
Yes, I think that the default choices are good ones for now. 

- Depending on what other things we ship via this mechanism, we may want to revisit the `dismissable` setting to bug our users more. This would be particularly true if we wanted to require something like our own block library plugin along with this theme - if it woudn't work out of the box on some level, then we can tighten things up a bit there.
- The auto-activate on install is one less click, but I think requiring that click from the end-user solidifies the experience of installing/activating a plugin as a "normal" behavior. 👍